src/execution_plan/ops/op_index_scan.c

```diff
@@ -43,7 +43,7 @@ static OpResult IndexScanInit(OpBase *opBase) {
 
 static inline void _UpdateRecord(IndexScan *op, Record r, EntityID node_id) {
 	// Populate the Record with the graph entity data.
-	Node n = {0};
+	Node n = NEW_LABELED_NODE(op->n->label, op->n->labelID);
```

Comment:
Dropping the label string and only retaining `labelID` would cause problems for OpCreate and OpMergeCreate, which will have a `GRAPH_UNKNOWN_LABEL` ID until the operation creates the appropriate schema.